examples/pylab_examples/log_demo.py

```diff
@@ -9,37 +9,41 @@
 import numpy as np
 import matplotlib.pyplot as plt
 
-plt.subplots_adjust(hspace=0.4)
+# Data for plotting
 t = np.arange(0.01, 20.0, 0.01)
 
+# Create figure
+fig, ((ax1, ax2), (ax3, ax4)) = plt.subplots(2, 2)
+
+# Note hspace is the amount of height reserved for white space between subplots
+# expressed as a fraction of the average axis height
+fig.subplots_adjust(hspace=0.5)
+
 # log y axis
-plt.subplot(221)
-plt.semilogy(t, np.exp(-t/5.0))
-plt.title('semilogy')
-plt.grid(True)
+ax1.semilogy(t, np.exp(-t/5.0))
+ax1.set(title='semilogy')
```

Comment:
No particular reason - in other pylab examples there are usually multiple keyword arguments being passed to ax.set but in this case only one (title). If way or the other is preferred, happy to update it.